src/components/app.js

```diff
@@ -88,6 +90,14 @@ const useStyles = makeStyles(() => ({
     process: {
         marginLeft: 18,
     },
+    child: {
+        width: '100%',
```

Comment:
why width 100% ? use flex-grow instead ?

Reply:
The bad effect of this is that it overflows the parent container and triggers flex-shrink on all items to fit

Reply:
@med-hedi